app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -46,6 +47,8 @@
 
     private SearchView searchView;
 
+    private TextView noSessionsView;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noSessionsView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630496/issues/source?bid=3233850&fileBranchId=3233897#l50)